index.html

```diff
@@ -14,18 +14,19 @@
 </head>
 <body class="background">
 
-	<nav class="text-center">
-	  <ul>
+	<nav class="text-center navbar">
+	  <!-- <ul> -->
```

Comment:
No need to comment stuff out. If you have confirmed that it is unnecessary, then just remove it entirely.